Simbody/include/simbody/internal/MobilizedBody_Translation.h

```diff
@@ -73,46 +73,81 @@ class SimTK_SIMBODY_EXPORT MobilizedBody::Translation : public MobilizedBody {
         (void)MobilizedBody::setDefaultOutboardFrame(X_BM); return *this;
     }
 
-    // This is just a nicer name for setting this mobilizer's generalized coordinates,
-    // which together constitute the vector from the F frame's origin to the M
-    // frame's origin, expressed in F.
-
-    // Set the topological default values for the initial q's.
+    /** This is just a nicer name for setting this mobilizer's topological
+    default generalized coordinates q, which together constitute the vector
+    p_FM from the F frame's origin Fo to the M frame's origin Mo, expressed
+    in F. */
     Translation& setDefaultTranslation(const Vec3& p_FM) {
         return setDefaultQ(p_FM);
     }
 
-    // Get the topological default values for the initial q's.
+    /** Get the topological default values for the initial q's. **/
     const Vec3& getDefaultTranslation() const {
         return getDefaultQ();
     }
 
-    // Set the current value of q's in the given State. Note that this is
-    // the *cross-mobilizer* translation, not location in the Ground frame.
-    void setMobilizerTranslation(State& s, const Vec3& p_FM) const {
+    /** Set the current value of q's in the given State. Note that this is
+    the _cross-mobilizer_ translation vector p_FM, not location in the Ground
+    frame. **/
+    void setTranslation(State& s, const Vec3& p_FM) const {
         setQ(s,p_FM);
     }
 
-    // Get the current value of the q's for this mobilizer from the given State.
-    const Vec3& getMobilizerTranslation(const State& s) const {
+    /** Get the current value of the q's for this mobilizer from the given
+    State. This is the _cross-mobilizer_ translation vector p_FM, not location
+    in the Ground frame. **/
```

Comment:
If you're willing to change `Set the` above to `The`, you could use `@copydoc` here instead of duplicating the comment.